### PR TITLE
Features/confirmed preference priority order #160361568

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/160361568

### CODE REVIEW

**Comment by akegan** [comment]
Here's what they look like now. 
image

The preference_order values for each of the preferences are:
HUD - 0
Test Custom Pref - 0
COP - 1
DTHP -2
NRHP - 3
Live/work - 4
Rent burdened/Assisted Housing - 5
Alice griffith - 6
ADHP - 7

File: app/javascript/components/supplemental_application/SupplementalApplicationPage.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ class SupplementalApplicationPage extends React.Component {
   render () {
     const { statusHistory, fileBaseUrl, application, availableUnits } = this.props
     const { confirmedPreferencesFailed, amis, amiCharts } = this.state
-
+    console.log('Application, ', application)
```
Comment (fddayan) [inline app/javascript/components/supplemental_application/SupplementalApplicationPage.js]: do we need this console.log?

Comment (akegan) [reply app/javascript/components/supplemental_application/SupplementalApplicationPage.js]: nope! good catch

**Comment by akegan** [comment]
@fddayan console log removed. Ready for review again

**Review by fddayan — APPROVED** [review]
The code looks good, review instructions are missing.

**Comment by akegan** [comment]
Thanks! review instructions added now. I'll wait to ask for QA until we get the semaphore build passing.

**Pull request merged**